docs/simulate/qvm_builder_code.ipynb

```diff
@@ -0,0 +1,224 @@
+    "\n",
+    "try:\n",
+    "    import cirq\n",
+    "    import cirq_google\n",
+    "except ImportError:\n",
+    "    print(\"installing cirq...\")\n",
+    "    !pip install --quiet cirq-google\n",
+    "    print(\"installed cirq.\")\n",
+    "    import cirq\n",
+    "    import cirq_google\n",
+    "\n",
+    "try:\n",
+    "    import qsimcirq\n",
+    "except ImportError:\n",
+    "    print(\"installing qsimcirq...\")\n",
+    "    !pip install --quiet qsimcirq\n",
+    "    print(f\"installed qsimcirq.\")\n",
+    "    import qsimcirq\n",
+    "\n",
+    "import time"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "p2JAfQa8gVSe"
+   },
+   "source": [
+    "## Create a **Quantum Virtual Machine**.\n",
+    "\n",
+    "Instantiate a `cirq.SimulatedLocalEngine` that uses the [Virtual Engine Interface](/cirq/simulate/virtual_engine_interface)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "pbHCUPLpq5WE"
+   },
+   "outputs": [],
+   "source": [
+    "# @title Choose a processor (\"rainbow\" or \"weber\")\n",
+    "processor_id = \"rainbow\"  # @param {type:\"string\"}\n",
+    "\n",
+    "# Instantiate an engine.\n",
+    "sim_engine = cirq_google.engine.create_default_noisy_quantum_virtual_machine(\n",
+    "    processor_id=processor_id, simulator_class=qsimcirq.QSimSimulator\n",
+    ")\n",
+    "print(\n",
+    "    \"Your quantum virtual machine\",\n",
+    "    processor_id,\n",
+    "    \"is ready, here is the qubit grid:\",\n",
+    "    \"\\n========================\\n\",\n",
+    ")\n",
+    "print(sim_engine.get_processor(processor_id).get_device())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "TJfN17frwo-0"
+   },
+   "source": [
+    "## **Create** a device-ready circuit."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "BYxi9xpXjJdI"
+   },
+   "source": [
+    "To learn how to create a device ready circuit, have a look at the [QVM Circuit Preparation](/cirq/simulate/qvm_basic_example) page."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "FCoKJhGri8lR"
+   },
+   "outputs": [],
+   "source": [
+    "# create your device ready circuit here!\n",
+    "your_circuit_name = cirq.Circuit()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "Zxv0RtJuhaof"
+   },
+   "source": [
+    "## **Execute** Your circuit on the Quantum Virtual Machine."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "bFjnNSqRZsFu"
+   },
+   "outputs": [],
+   "source": [
+    "# @title Enter the name of your device ready circuit and execute it on the Quantum Virtual Machine\n",
+    "circuit = your_circuit_name  # @param\n",
+    "\n",
+    "reps = 3000\n",
+    "start = time.time()\n",
+    "results = sim_engine.get_sampler(processor_id).run(circuit, repetitions=reps)\n",
```

Comment:
We either need to comment this out, add output for it, or add a measurement (ideally to a qubit both on weber and rainbow) in the circuit (see above).